# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorANGLE.h
# Commit: d7be4e6bb7f1
# Full Hash: d7be4e6bb7f108cca7e643fdb47ffad1bded1775
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2019-06-26 21:52:21
# Description:
#   Bug 1559681 - Repair device-reset handling in RenderCompositorANGLE. r=sotaro
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35882
# ==============================================================================

diff -r 88184dadb61a -r d7be4e6bb7f1 gfx/webrender_bindings/RenderCompositorANGLE.h
--- a/gfx/webrender_bindings/RenderCompositorANGLE.h	Wed Jun 26 15:34:14 2019 +0000
+++ b/gfx/webrender_bindings/RenderCompositorANGLE.h	Wed Jun 26 06:03:23 2019 +0000
@@ -35,8 +35,7 @@
   static UniquePtr<RenderCompositor> Create(
       RefPtr<widget::CompositorWidget>&& aWidget);
 
-  explicit RenderCompositorANGLE(RefPtr<widget::CompositorWidget>&& aWidget,
-                                 gl::GLLibraryEGL*);
+  explicit RenderCompositorANGLE(RefPtr<widget::CompositorWidget>&& aWidget);
   virtual ~RenderCompositorANGLE();
   bool Initialize();
 
@@ -68,7 +67,6 @@
   bool SutdownEGLLibraryIfNecessary();
   RefPtr<ID3D11Query> GetD3D11Query();
 
-  RefPtr<gl::GLLibraryEGL> mEgl;
   EGLConfig mEGLConfig;
   EGLSurface mEGLSurface;
 
